src/Containers/FileUpload/BottomPanel.tsx

```diff
@@ -0,0 +1,30 @@
+import React,{forwardRef} from 'react'
+import {Container} from './Container'
+
+interface IBottomPanelProps{
```

Comment:
Why is this not extending HTMLDivElemt so all the props don't need to be retyped??